to_do_list/src/Components/SaveButton.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Fab } from "@material-ui/core";
+import SaveAltIcon from "@material-ui/icons/SaveAlt";
+import config from '../config';
+import storageManager from '../StorageManager';
+
+function SaveButton(props) {
+  const saveToLocalStorage = () => {
+    storageManager.save(config.STORAGE_KEY, props.data);
+  }
+
+  return (
+    <Fab color="secondary" onClick={saveToLocalStorage}>
```

Comment:
i didn't even understand what this button was until i checked the code.
atleast add some kind of alt so i can hover over the button to see what it's doing.